src/js/components/homepage/features/ImageCarousel.jsx

```diff
@@ -217,11 +217,11 @@ export default class ImageCarousel extends React.Component {
                     key={image.key || image.src}
                     aria-hidden={this.state.page !== index + 1}
                     tabIndex={-1}>
-                    <img
-                        className="feature-carousel-image__image"
-                        srcSet={image.srcSet}
-                        src={image.src}
-                        alt={image.alt} />
+                    <picture className="feature-carousel-image__image">
+                        <source srcSet={image.srcSet} type="image/webp" />
+                        <source srcSet={image.src} type="image/png" />
+                        <img src={image.src} alt={image.alt} />
+                    </picture>
```

Comment:
Very clever! Did you check IE, which doesn't officially support < picture > at all? https://caniuse.com/?search=picture

Reply:
Yes, it was working in IE 11 for me so maybe it's able to fallback to the `<img>`?

Reply:
If you're able to pull it up locally in IE to double check, that would make me feel better! 😅